.gitignore

```diff
@@ -1,5 +1,6 @@
 # Directories
 /ewww
+/bin
```

Comment:
@rgadon107 We actually want to keep `/bin` in the repo. We'll strip it out when it comes time to do a formal release.

Reply:
@hellofromtonya Oh, I didn't realize that. Will remove.